lib/rollout.rb

```diff
@@ -180,21 +180,21 @@ def active_in_group?(group, user)
     f = @groups[group.to_sym]
     f && f.call(user)
   end
+  
+  def multi_get(*features)
+    feature_keys = features.map{|feature| key(feature) }
+    feature_strings = @storage.mget(*feature_keys)
+    collection = []
+    feature_strings.each_with_index do |string, index|
+      feature = features[index]
+      collection << load_feature(feature, string)
+    end
+    collection
```

Comment:
this might be cleaner if you use `feature_strings.map.with_index`. Then you can get rid of the collection variable